pkg/interface/chat/src/js/components/lib/message.js

```diff
@@ -167,10 +167,10 @@ export class Message extends Component {
       let color = "#000000";
       if (contact) {
         name = (contact.nickname.length > 0)
```

Comment:
Cleaner? Idk, you can do \`~${props.msg.author}\` if you use backtick quotes like that, I guess.

Reply:
Hmm, could wee a variable that prepends the sig? This would make it easier to QA.

Reply:
(Idk why my phone made it do this a line too high.) Honestly this is fine as is imo but I’ll wait and see what the other reviewers think too.

Reply:
I prefer Matilde's version of this as well